modules/oadp-installing-dpa.adoc

```diff
@@ -80,6 +82,7 @@ spec:
 ----
 <.> The `openshift` plug-in is mandatory.
 <.> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
+<.> Specify the node selector to be supplied to Restic podSpec.
```

Comment:
```suggestion
<.> Specify the node selector to be supplied to Restic `podSpec`.
```
Not a merge blocker. Please address throughout the PR.

Reply:
Not a blocker:
After GA, I recommend checking your customer portal build. There are instances where the portal does not render `<.>`, autonumbering callouts correctly. Someone is working on updating the docs repo guidelines to call this out. Not urgent, just an FYI.